tasking/pulp/tasking/storage.py

```diff
@@ -0,0 +1,138 @@
+import errno
+import os
+import shutil
+import stat
+
+from celery import task
+from django.conf import settings as pulp_settings
+
+
+def _working_dir_root(worker_name):
+    """Return the path to the working directory of a worker.
+
+    :param worker_name:     Name of worker for which path is requested
+    :type  worker_name:     basestring
+
+    :returns: working_directory setting from server config
+    :rtype:   str
+    """
+    return os.path.join(pulp_settings.SERVER['working_directory'], worker_name)
+
+
+def create_worker_working_directory(worker_name):
+    """Create a working directory for a worker.
+
+    Create a working directory inside the working_directory as specified in the setting file.
+
+    :param worker_name:     Name of worker that uses the working directory created
+    :type  worker_name:     basestring
+
+    :returns: Path to the working directory for a worker
+    :rtype:   str
+    """
+    working_dir_root = _working_dir_root(worker_name)
+    os.mkdir(working_dir_root)
```

Comment:
Not suppressing it sounds fine. It sounds like that is what the code that calls this function wants too.